sensor_msgs/Image raw_image
sensor_msgs/Image undistorted_image
